presentation/scripts/lab1-1.sh

```diff
@@ -22,11 +22,4 @@ doit curl -s localhost:32768
 comment "Clean up container"
 doit docker rm -f ${CID}
 
-# Push the image to the registry so Kubernetes wil find it
-comment "Push the image to the IBM Cloud registry"
-doit docker push ${IMAGE_NAME}:v1
-
-comment "Verify it's there"
-doit bx cr images
-
```

Comment:
If we're dropping cr also take out this stuff. 
https://github.com/duglin/kube101/blame/ab68da188a4574b2f612e0c2f6e22dbf9c3f4a98/presentation/scripts/README.md#L5

Reply:
done